src/Tizen.Applications.PackageManager/Tizen.Applications/PackageFilter.cs

```diff
@@ -45,6 +47,28 @@ public PackageFilter(IDictionary<string, bool> filter)
             _filter = filter;
         }
 
+        /// <summary>
+        /// The constructor with specific filters. Using this will filter out the installed packages which do not meet the filter criteria.
+        /// </summary>
+        /// <param name="stringFilter">Package filters using string values.</param>
```

Comment:
it would be better to have some info about what to put in the dictionary, on the other hand there is the Keys class

Reply:
I added remarks.